.ci.yaml

```diff
@@ -338,6 +338,7 @@ targets:
         {"download_emsdk": true}
       add_recipes_cq: "true"
       build_host: "true"
+      mac_model: "Macmini8,1"
```

Comment:
I think this has to go into the json files where we specify the dimensions. Does this work?

Reply:
https://cs.opensource.google/flutter/cocoon/+/main:app_dart/lib/src/model/ci_yaml/target.dart;l=40

This does get run, but it'll only change the orchestrator vm. This is useful as the spawned tasks should inherit the properties from the orchestrator.